File: C/2000-2999/2949.c
#include <stdio.h>
#define max 100

int main (void){
  int vez, hob = 0, hum = 0, elf = 0, ana = 0, mag = 0;
  char lt, nome[max];

  scanf("%d", &vez);

  for (int i = 0; i < vez; i++){
    scanf("%s %c", &nome, &lt);

    switch(lt){
        case 'X':
            hob += 1;
            break;
        case 'M':
            mag += 1;
            break;
        case 'H':
            hum += 1;
            break;
        case 'A':
            ana += 1;
            break;
        case 'E':
            elf += 1;
            break;
    }

  }
  printf("%d Hobbit(s)\n", hob);
  printf("%d Humano(s)\n", hum);
  printf("%d Elfo(s)\n", elf);
  printf("%d Anao(s)\n", ana);
  printf("%d Mago(s)\n", mag);

  return 0;
}

File: Python/2000-3000/2949.py
# -*- coding: utf-8 -*-

def main():
    n = int(input())
    anao = 0
    elfo = 0
    humanos = 0
    magos = 0
    xHobbits = 0


    for c in range(0, n):
        nome, raca = input().split()

        if raca == 'A':
            anao += 1
        elif raca == 'E':
            elfo += 1
        elif raca == 'H':
            humanos += 1
        elif raca == 'M':
            magos += 1
        elif raca == 'X':
            xHobbits += 1
        
    print('%i Hobbit(s)' %(xHobbits))
    print('%i Humano(s)' %(humanos))
    print('%i Elfo(s)' %(elfo))
    print('%i Anao(s)' %(anao))
    print('%i Mago(s)' %(magos))


main()